recipes/cfm-id/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set CONDA_BOOST = "1.66" %}
+package:
+  name: cfm-id
+  version: "33"
+
+source:
+  #TODO needs to be moved to cargo port since snapshots are removed by sourceforge
+  url: https://sourceforge.net/code-snapshots/svn/c/cf/cfm-id/code/cfm-id-code-r33-cfm.zip
```

Comment:
Could you adapt the url when the correct one is available in cargo port: https://github.com/galaxyproject/cargo-port/pull/153#issuecomment-465977968

Reply:
done